### DISCUSSION TITLE
Disabled all alerts based on a condition?

### DISCUSSION THREAD

**Discussion by ricardodacosta** [timelyart/Kairos#112] [Q&A]
Is it possible to automatically disable all alerts based on a certain condition such as (if 4 alerts triggered, stop all alerts)

Thank you!

**Comment by timelyart** [comment]
Currently, no. Can you sketch me a scenario wherein this would be useful?

It wouldn't be too hard to add but bear in mind that the quicker you want the stop the alerts after the condition is met the lower the interval would to run Kairos, e.g. if you run Kairos once an hour, well the alerts will only be stopped once an hour.
